_episodes/11-software-project.md

```diff
@@ -55,67 +55,59 @@ to fix and build on top of the existing code during the course.
 ## Downloading Our Software Project
 
 To start working on the project, you will first create a fork of the software project repository
-from GitHub within your own GitHub account
-and then obtain a local copy of that project (from your GitHub) on your machine.
+from GitLab within the workshop's GitLab group
```

Comment:
Just checking: is this intentional? I would imagine that each participant creates their own copy. Generally, it might be confusing to introduce GitLab groups here while it might not be entirely clear to everyone if it means that they have to do something indvidiually.

Reply:
Yes this is intentional. We will have a dedicated GitLab group on TNOs gitlab instance. That is why I ask to add the name to it ;)